azure-pipelines.yml

```diff
@@ -172,7 +172,7 @@ stages:
 
         - powershell: ./eng/scripts/InstallProcDump.ps1
           displayName: Install ProcDump
-        - powershell: ./eng/scripts/StartDumpCollectionForHangingBuilds.ps1 $(ProcDumpPath)procdump.exe artifacts/log/$(_BuildConfig) (Get-Date).AddMinutes(25) dotnet, msbuild
+        - powershell: ./eng/scripts/StartDumpCollectionForHangingBuilds.ps1 $(ProcDumpPath)procdump.exe artifacts/log/$(_BuildConfig) (Get-Date).AddMinutes(25) dotnet, msbuild, devenv, vbcscompiler, xunit.console, xunit.console.x86
```

Comment:
Not sure we need anything other than the devenv / xunit bits. Maybe we'd need Roslyn's OOP process. I guess I just haven't seen value in the existing dumps for `dotnet` / `MSBuild`

Reply:
> I guess I just haven't seen value in the existing dumps for dotnet / MSBuild

Yeah that tracks. I'd be down to remove those and the vbcscompiler as any hang that DID exist in those would likely be solidly out of our purvey.